drivers/ethernet/Kconfig

```diff
@@ -32,6 +32,15 @@ config ETH_QEMU_IFACE_NAME
 	  script from net-tools project can be used to create the network
 	  interface.
 
+config ETH_QEMU_EXTRA_ARGS
```

Comment:
I think we should consider merging this and `NET_QEMU_USER_EXTRA_ARGS` for a consolidated Kconfig option.. Both are used to convey similar type of information in different scenarios.. Thoughts ?

Reply:
Well, atm they are mutually exclusive (cannot exists at the same time) so is no big issue here. Also you might want to have different options for them so switch between user/tap mode would be cumbersome.

Reply:
Regarding the extra arguments, and the help text below, then there is no way for the user to know if he should provide a `;`-separated (CMake style),  `,`-separated, or `' '`-separated list in this field.
From the CMake code, I can see the only valid separator to use when providing multiple args is `,`.

Would it be possible to syntax validate the extra args, before passing it to QEMU ?

Reply:
> From the CMake code, I can see the only valid separator to use when providing multiple args is `,`.
> 
> Would it be possible to syntax validate the extra args, before passing it to QEMU ?

I can add more text to the help string. I think we can leave syntax validation to qemu, seems pointless to do that in both in cmake file and qemu, when the qemu does that anyway.